frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/hooks/dristi/useGetHearings.js

```diff
@@ -1,14 +1,15 @@
 import { useQuery } from "react-query";
 import { DRISTIService } from "../../services";
 
-function useGetHearings(data, params, keys, enabled) {
+function useGetHearings(data, params, keys, enabled, refetchInterval = 0) {
```

Comment:
default value should be false for [refetchInterval](https://tanstack.com/query/latest/docs/framework/react/reference/useQuery)